src/arrayMethodSort.js

```diff
@@ -4,8 +4,34 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    if (compareFunction === undefined) {
+      for (let i = 0; i < this.length - 1; i++) {
+        for (let j = i + 1; j < this.length; j++) {
+          const a = '' + this[i];
+          const b = '' + this[j];
+          const change = this[i];
+
+          if (a > b) {
+            this[i] = this[j];
+            this[j] = change;
+          }
+        }
+      }
```

Comment:
Create a separate function and inside compare values as strings and use this function as a default value for compareFunction